### ISSUE TITLE
Remove semantic <section>s from footer columns

### ISSUE THREAD

**Issue by cjg89** [UCF/UCF-WordPress-Theme#76]
**Describe the bug**

`<section>` elements in our footer template part cause non-distinguishable landmark warnings to be thrown due to a lack of a label on each section.

**Steps to Reproduce**

n/a; just look at the template part file

**Expected Behavior**

All non-unique landmarks defined in the theme should have labels assigned to them.  Because we can't define a specific enough label for these footer columns, we should remove the landmark association with them.

**Possible Fix**

Replace these sections with divs or an alternate semantically-appropriate element without the landmark association.

**Your environment:**

(all)

**Additional context**

n/a

**Comment by jmbarne3** [comment]
Resolved in #79
